### PR TITLE
Impelentation completed

### PR DESCRIPTION
**Pull request by PierwszyKrytykRP**

### CODE REVIEW

File: module1/homework/calculate/calculate.hpp [hunk 1]
```diff
@@ -3,5 +3,24 @@
...
+
+    if("add" == command)
+        result = first + second;
+    else if("subtract" == command)
+        result = first - second;
+    else if("multiply" == command)
+        result = first * second;
+    else if("divide" == command)
+    {
+        if(0 == second)
```
Comment (Kasperice) [inline module1/homework/calculate/calculate.hpp:17]: I think ```if (second == 0)``` will be more clear, but as you can see ```if (0 == second)``` also works

Comment (Kasperice) [reply module1/homework/calculate/calculate.hpp:17]: It also applies for all of your conditionals

Comment (PierwszyKrytykRP) [reply module1/homework/calculate/calculate.hpp:17]: I consider `"divide" == command` to be safer. I won't accidentally assign value of right side to the left side of comparison if i forget one `=`. I would get compile error

**Comment by ziobron** [comment]
rid:1429 - calculate +5XP

**Comment by ziobron** [comment]
rid:1119 - punktualność +2XP

**Pull request closed without merging**